gc/base/IndexableObjectScanner.hpp

```diff
@@ -91,7 +108,26 @@ class GC_IndexableObjectScanner : public GC_ObjectScanner
 		GC_ObjectScanner::initialize(env);
 	}
 
-public:
+	/**
+	 * Get the current element of the array
+	 */
+	MMINLINE fomrobject_t *getScanPtr() { return _scanPtr; }
+
+	/**
+	 * Get the address of the next element in the array.
+	 * @return NULL if there are no more elements
+	 */
+	MMINLINE fomrobject_t *
+	next()
```

Comment:
perhaps nextElement (or getNextElement), just to avoid confusion with getNextSlotMap?

Reply:
Changed to `nextIndexableElement()`